terraform/main.tf

```diff
@@ -113,16 +111,14 @@ locals {
       osdisk_size_controller  = "150"
       num_builders_x86        = 0
       num_builders_aarch64    = 0
-      # 'priv' and 'dev' environments use the same binary cache signing key
-      binary_cache_public_key = "ghaf-infra-dev:EdgcUJsErufZitluMOYmoJDMQE+HFyveI/D270Cr84I="
-      binary_cache_url        = "https://ghaf-binary-cache-${local.ws}.${azurerm_resource_group.infra.location}.cloudapp.azure.com"
-      ext_builder_machines = [
-        "ssh://remote-build@builder.vedenemo.dev x86_64-linux /etc/secrets/remote-build-ssh-key 32 3 kvm,nixos-test,benchmark,big-parallel - -",
-        "ssh://remote-build@hetzarm.vedenemo.dev aarch64-linux /etc/secrets/remote-build-ssh-key 40 3 kvm,nixos-test,benchmark,big-parallel - -"
-      ]
-      ext_builder_keyscan = ["builder.vedenemo.dev", "hetzarm.vedenemo.dev"]
+      binary_cache_public_key = "priv-cache.vedenemo.dev~1:FmJGfAkx+2fhqpzHGT/V3M35VcPm2pfkCuiTo8xQD0A="
+      binary_cache_url        = local.binary_cache_url_common
+      ext_builder_machines    = local.ext_builder_machines
+      ext_builder_keyscan     = local.ext_builder_keyscan
     }
     dev = {
+      builder_sshkey_id       = "ext"
+      persistent_id           = "prod"
```

Comment:
maybe incorrect "prod" persistent_id  in dev environment ?

Reply:
That's intentional. As outlined in https://ssrc.atlassian.net/wiki/spaces/SP/pages/1357250598/Ghaf-infra+Environments and also in this PR's description, we want the `dev` and `prod` to share the same binary cache storage and nix signing key.